recipes/vtk/all/conandata.yml

```diff
@@ -0,0 +1,10 @@
+sources:
+  "9.0.1":
+    sha256: 9229bf40d4d9a9e4d8a118149cff031b5199cd2cd20df6c41e5da7cd1c7ad23d
+    url: https://github.com/Kitware/VTK/archive/v9.0.1.tar.gz
+patches:
+  "8.2.0":
```

Comment:
`8.2.0` is not added by this repo.
Did you mean `"9.0.1"`?

Reply:
It is correct.
It should be removed just before merge to cci, if we decide only provide 9.0.1 (and not providing 8.2.0 in first version of this recipe)

Reply:
Done